### ISSUE TITLE
Table of Contents

### ISSUE THREAD

**Issue by emcconville** [IM6-Documentation/im6-usage-unify-markdown#1]
All documents should have a uniformed table of contents. Ideally an unordered list without image bullets

Example 

    - Section
       - sub Section

**Comment by KurtPfeifle** [comment]
I don't think that the *Markdown* source files should have a ToC (at least not a hand-written one).

Pandoc can auto-generate a ToC from the headlines (any depth we want), and CSS can style it (any way we like).

However, all documents should have enough of a *structure* to support the auto-generation of a ToC.

**Comment by emcconville** [comment]
Great point. Updating this task
